ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -146,6 +147,10 @@ export class FrontstageDef {
     return this._contentGroup;
   }
 
+  public get activeToolEmptyMessage(): string | undefined {
```

Comment:
Let's mark everything else except config part as `@internal`
```suggestion
  /** @internal */
  public get activeToolEmptyMessage(): string | undefined {
```

Reply:
This function has been removed.